etc/kayobe/kolla/config/monasca/sv-b16-u23/agent_plugins/rabbitmq.yaml

```diff
@@ -0,0 +1,21 @@
+{% raw %}
```

Comment:
I think you'll need to rename this to sv-b16-u21.

Reply:
doh